.github/workflows/python-test.yml

```diff
@@ -37,12 +37,17 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install and configure Poetry
-        uses: snok/install-poetry@v1
-        with:
-          version: 1.4.0
-          virtualenvs-create: true
-          virtualenvs-in-project: true
-          installer-parallel: true
+        env:
+          POETRY_VERSION: 1.4.0
+        run: |
+          pip install poetry==$POETRY_VERSION
+          poetry config virtualenvs.create true
+          poetry config virtualenvs.in-project true
+          poetry config installer.parallel true
+      - name: Install poetry version plugin
+        run: |
+          pip install -e python/plugin/poetry-version-plugin 
```

Comment:
Can we install the plug-in with poetry plugin
$ poetry plugin add poetry-version-plugin